events/2024-11-in-person/index.qmd

```diff
@@ -0,0 +1,142 @@
+| 13:00 | Personal goals and perspectives                               |
+| 14:00 | ON-LiMiT and DP-Next projects presentations and brainstorming |
+| 15:00 | :coffee: Coffee break                                         |
+| 15:15 | Continue from previous session                                |
+| 16:30 | Wrap Up                                                       |
+| 17:30 | Dinner and social time?                                       |
+
+### Nov 28
+
+| Time  | Item                                   |
+|-------|----------------------------------------|
+| 10:00 | Overview of progress and current state |
+| 11:45 | Lunch                                  |
+| 12:30 | Status update with Annelli             |
+| 14:00 | :coffee: Coffee break                  |
+| 14:15 | Roadmap, deliverables, and milestones  |
+| 16:30 | Wrap Up                                |
+| 17:30 | Team dinner                            |
+
+## Sessions
+
+All brainstorming sessions start with some dedicated individual time to
+think and write down thoughts, then we will discuss them all together.
+
+Maybe use stickies to write down thoughts and then discuss them?
+
+### Align understandings about the current state of Seedcase
+
+**Aim**: Make sure we are all aligned with what we are building and why
+and for who and what it might look like.
+
+-   It seems like there is a bit of a drift in understanding of what
+    we're doing, why, and how. So this is some dedicated time to assess
+    how we all understand things and take some time to update and
+    discuss it.
+-   Discuss and brainstorm how Seedcase will look like in the end and
+    how users might interact with it.
+-   Start broad with all of Seedcase, then slowly focus in on Sprout and
+    it's subcomponents.
+
+### Personal goals and perspectives of what we create
+
+**Aim**: Brainstorm, share, and discuss our own personal goals and
+perspectives on what we create and do when working on Seedcase. This is
+mainly to help us understand each other better and to help us support
+each other in working with our strengths, interests, and passions.
+
+-   Write out and share how we each would like to see Seedcase develop
+    in the next few years *and* what we would like to work on over that
+    time.
+-   Brainstorm what we can do to support and encourage these goals and
+    perspectives in our work.
+
+### Overview of progress and current status
+
+**Aim**: Discuss where we are at and what we've done in order to connect
+with previous sessions.
+
+-   What we've achieved so far and how close we are to an minimum viable
+    product (MVP).
+
+-   Discuss any barriers and challenges? Both individually and as a
+    team.
+
+    -   Brainstorm ways we can work to resolve them?
+
+-   How does the progress and current status, as well as the challenges
+    and barriers, align with the goals and perspectives we just
+    discussed?
+
+### Roadmap, deliverables, and milestones
```

Comment:
Are deliverables Sprout, Propagate, Flower, and Garden? Or does each split in Sprout count as its own deliverable?

Reply:
I would say the first upload to PyPI would be a deliverable, so that could be after `lib` is done. But we can decide on this :relaxed: